### ISSUE TITLE
kubeflow 1.3 new notebook server get gpu value as string

### ISSUE THREAD

**Issue by bohau** [canonical/bundle-kubeflow#390]
I click button to create a notebook server with GPU asociated, but i got the error as below.

**Error**
[422] Notebook.kubeflow.org "test-jupyter" is invalid: spec.template.spec.containers.resources.limits.nvidia.com/gpu: Invalid value: "integer": spec.template.spec.containers.resources.limits.nvidia.com/gpu in body must be of type string: "integer" http://xxxxx/jupyter/api/namespaces/admin/notebooks

**Comment by amitk75** [comment]
> I click button to create a notebook server with GPU asociated, but i got the error as below.
> 
> **Error** [422] Notebook.kubeflow.org "test-jupyter" is invalid: spec.template.spec.containers.resources.limits.nvidia.com/gpu: Invalid value: "integer": spec.template.spec.containers.resources.limits.nvidia.com/gpu in body must be of type string: "integer" http://xxxxx/jupyter/api/namespaces/admin/notebooks

@bohau Were you able to resolve this? I am also facing same issue. 

Thanks in advance,
Amit

**Comment by gcheron** [comment]
I am also facing same issue! Were you able to resolve this?

**Comment by DnPlas** [comment]
Hi @bohau, thanks for reporting this. Our latest release should have the fix for this issue (as discussed in #408), do you mind letting us know if you can reproduce the issue using `kubeflow latest/stable`?

**Comment by DnPlas** [comment]
Hi @bohau. I tried reproducing the error using our latest release, as mentioned in my last comment, and this is in fact a bug that's not solved by the upgrade. I've labeled this issue as a duplicate of [#26](https://github.com/canonical/notebook-operators/issues/26) as that one is filed in `canonical/notebook-operators`, which is more appropriate. I'll close this issue and continue the discussion over there.
